### DISCUSSION TITLE
[Feature Request] Add number of pages.

### DISCUSSION THREAD

**Discussion by Dtrieb** [astubenbord/paperless-mobile#200] [Feature Requests]
Could you add the number of pages of the document p.e. on the preview or metadata?

**Comment by astubenbord** [comment]
It would be possible but it would also require to always download the pdf file when opening the details view as the number of pages is not provided by the API. 

Also, since this is a mobile app, I don't necessarily want to load data via network that is not required, especially for larger documents.

**Comment by Dtrieb** [comment]
Okay I understand.
Possible as optional option.
